src/Reservation.php

```diff
@@ -703,7 +706,7 @@ public function showForm($ID, array $options = [])
         } else {
             $resa->getEmpty();
             $options = Planning::cleanDates($options);
-            $resa->fields["begin"] = date("Y-m-d H:i:s", strtotime($options['begin']));
+            $resa->fields["begin"] = !empty($options['begin']) ? date("Y-m-d H:i:s", strtotime($options['begin'])) : date('Y-m-d H:00:00');
```

Comment:
We try to reuse only one source of "current time" to make sure it is always the same time, which is set in the session.

```suggestion
            $resa->fields["begin"] = !empty($options['begin']) ? date("Y-m-d H:i:s", strtotime($options['begin'])) : date('Y-m-d H:00:00', strtotime(Session::getCurrentTime()));
```